aws/autogluon/autogluon_airline.ipynb

```diff
@@ -2,7 +2,78 @@
  "cells": [
```

Comment:
<p>You mentioned this data is also available in us-east region?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>